python/cudf/tests/test_string.py

```diff
@@ -322,8 +342,6 @@ def test_string_cat(ps_gs, others, sep, na_rep):
     if isinstance(pd_others, pd.Series):
         pd_others = pd_others.values
     expect = ps.str.cat(others=pd_others, sep=sep, na_rep=na_rep)
-    if isinstance(others, pd.Series):
```

Comment:
With new pd.Series handling which I'm doing in the code, we don't need these two lines which existed before.